### DISCUSSION TITLE
Deleting folders doesn't work in new outlook version

### DISCUSSION THREAD

**Discussion by thomastaylor747** [microsoft/mfcmapi#595] [Q&A]
Hello, 

I'm using the latest version of MFCMapi.exe 64 bit version as my outlook is 64 bit.

Microsoft Outlook Version 1.2023.616.100 (Production).
Client Version is 20230623002.12 .

Deleting the RSS Subscriptions folder doesn't work it allows me to delete it on MFCMapi but it doesn't delete from Outlook. If I revert to the old outlook style it works but when I toggle the "New Outlook" view it reappears.

Is this something you're aware of or is there a fix for it?

Thanks!

**Comment by stephenegriffin** [comment]
"New Outlook" is a different client which doesn't use MAPI at all. MFCMAPI doesn't work with new outlook
